### ISSUE TITLE
Connect the rationales provided via config file with the csv output

### ISSUE THREAD

**Issue by adhilto** [cisagov/ScubaGear#1289]
# 💡 Summary #
https://github.com/cisagov/ScubaGear/pull/1281 adds new columns for optional documentation purposes (e.g., "Non-Compliance Reason", "Remediation Completion Date"). The rationale provided when users omit a control (and optionally, the omit expiration date value) could be used to auto-populate these values, when provided.

## Motivation and context ##
Nice to have feature to make documentation more convenient.

## Implementation notes ##
Blocked by https://github.com/cisagov/ScubaGear/pull/1281.

## Acceptance criteria ##
- [ ] The omit rationale is connected to the csv output, when provided
- [ ] The omit expiration date is connected to the csv output, when provided

**Comment by FollyBeachGurl** [comment]
This issue is no longer relevant because the ScubaResult.json file has that data and the CSV isn't really used for anything.
